apps/site/public/types/modules/graph/0.3/schema/aggregation-operation.json

```diff
@@ -1,87 +1,125 @@
 {
+  "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "https://blockprotocol.org/types/modules/graph/0.3/schema/aggregation-operation",
-  "title": "AggregationOperation",
+  "title": "Aggregation Operation",
   "type": "object",
   "properties": {
-    "entityTypeId": {
-      "type": "string"
-    },
-    "pageNumber": {
-      "type": "number"
-    },
-    "itemsPerPage": {
-      "type": "number"
-    },
-    "multiFilter": {
+    "operation": {
+      "type": "object",
       "properties": {
-        "filters": {
-          "items": {
-            "anyOf": [
-              {
-                "properties": {
-                  "field": {
-                    "type": "string"
-                  },
-                  "operator": {
-                    "$ref": "#/$defs/FilterOperatorRequiringValue"
+        "multiFilter": {
+          "type": "object",
+          "properties": {
+            "filters": {
+              "type": "array",
+              "items": {
+                "anyOf": [
```

Comment:
I know this was here previously but I _think_ we actually want `oneOf`, an individual item can't be both (or at least shouldn't be)

Reply:
I was not sure, if `oneOf` implies, that all elements in the array have to be the same. It appears it has no effect

Reply:
Neither was I, I had to google it before writing this comment 😁

Reply:
Fixed in [Apply suggestions from Code Review](https://github.com/blockprotocol/blockprotocol/pull/1034/commits/2e5bd9dc964ed1713eb1ec846335a89862f8e4a5)